deployments/properties/azure-kubernetes-service-kustomize.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Build and Deploy an app to AKS with Kustomize",
```

Comment:
This name looks big, can we have smaller name for eg: `Deploy to AKS with Kustomize `

Reply:
Shortened